### PR TITLE
Fix FAQ link.

### PR DESCRIPTION
**Pull request by sseering**
¯\\_(ツ)_/¯

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,12 +25,12 @@ If you're interested, please see the [announcement video here](https://youtu.be/
   * **NEW** Delete shadercache and compatdata for whichever games you select
 * **NEW** Full CLI mode
 
-Look below for common questions and answers, or go check out my [YouTube Channel](https://www.youtube.com/@cryobyte33) 
+Look below for common questions and answers, or go check out my [YouTube Channel](https://www.youtube.com/@cryobyte33)
```
Comment (rvalitov) [inline README.md:28]: These lines are identical?

**Review by rvalitov — COMMENTED** [review]
All lines except the last one are identical? So there's no need to add them to the PR?

**Comment by rvalitov** [comment]
Fixes #33

**Comment by CryoByte33** [comment]
That's what I get for using a test repo, and apparently my IDE needs tuning. Thank you!

**Pull request merged**